rfc8446.xml

```diff
@@ -7693,1913 +8549,111 @@ kazu@iij.ad.jp</t>
-Q7ApnEyj28NxUruslfa8oxdZPQUuc6j8opQkH+aNudVG0OUXzZuTFYxru2JW
-a+7EJgF+O+5rLjKV5KV0VekIvLegKMOrXAE07xWdlbgEF1HuKASsQc/rCZCM
-/zTHEAiY4bhkk8xKsqlkuipRjOn20/0o1UCtxFwzrZKfEfoDOyJ+5lPVsqSe
-uUv1RgdpHad8aEyqEXwtyYmOgBrtTDGigBVAnhjazbGNZdu+2V7DTyepO8uL
-STLlGWkv/inLvxe9phIAnNjMfeo8l4YcPXoUNXv1+HUQ9ES6Itlv3N8Mbtrw
-BW4ndioR/DKC+F+O2zdArQS+rz3kSgTeht3VLn2hUUry9uB6/Vu+CxlnqCmh
-msfoNgAH6+ykQnFIMTCyt3cEvUYjrLqhHOUuNMxgtAqv6vdnqStn5L+W3agg
-VPVVu11h/l3orScQEiiXdhK5z4gLQZYsVJMHrgnS0KYcYHjiXELwLl7XN8u2
-nms28/WSMu0pYY9DcSgzJiOaMuZ5AyzikVrnDin6woijMdsVAFtURKPsTb17
-CSpqYCYsW87PeKUW2a9yCPpaPFVB8R/rhjCocN3c8FzP2SJYIkP0oiBWuKpR
-NbqMDSod33alZeima0Pb0znDxS9zRQBrZXl9vFFBWHcNFOqVsKJpm1jbZb2O
-wvu/op4gqj15HaR5gWmRzourNj81JMJK5k635fMQGscK/3tHdDCpj2es/ot/
-dEck7CA9H4+dl+79eOJ/B8P8c6oCHk5W/hx0O7NNukTK056oh33FBfjNZcHM
-EZjfQ1nxxwJUbqsZFHAJrrtiFKukossGiM7mr1IqNrJEkHPeLAW0ap4rlHLx
-xEuAzqxuLPMfLAsmGNjOnPtL9QQ0IFfCxRBrFD+7aqMhtVy8oc9uFnrVW/Tb
-+QNGmvfhCnTYtVAavWCaiI2izl0MvE1LCTR+NDDxi+E8FlNT3hVtc8Ns8VSv
-PoO8GJqZ0O/xWJ6Lg1tctqzKwSf2lj1t0YZu1XBXzC+0GMXirPHT0K7IbB6I
-TFnej7jIC49/zjsFjvcsknM7HZsHvOLQ/TAd2zil9HqEpWETuU7ueVcxkoVm
-LWEiI93bB6cu9Q0uROWhX1l5ddbtLkouGeXOZIZRCdSkpf60PZwCNFYIA8bZ
-e+q3RW8dC/I3+gJh74m3xJpiuMpfjNfuSc0TfwIlAbZ+JL/p0iywJfKaEk0s
-ZBp80Q8pZ9Gc2MQnyAfGu+odobdgtldtttOtXaq8YO8EhaqByyXTKNcbwjLd
-3oW2/FiBl6cNHO6ygX0gdfj6vYiX0jJo1hX7kwzZdXArc/XEYvW2Xb5FuFG8
-j9Sf+NbaXEiWUmtInMAZIozSaE3WN556yNdsOvjN1fbqHG6kUMTrrIbAcVMO
-zF/KDRPqC1lYoAWmzFq98Pli0MQv8+/0G4aPZ7akFOEoVGcSoRO/p0I8bwgC
-AxUm6ZqTIIi4GW/AahIsNHeIrVmULnv4comccQjThWaiTRgv3vrqfPF6qwqY
-4ToKceQnQGmiGXnxp+qP1yh4/fAJRaSQeEBBxN3EAPxIvOw32qdmTkQB+kP6
-43H8Y0ELEL/L30t0zHv8V+XTKlnP+FOPB9ql7KP8qcxN5DkV1prvqJLAGtG7
-L5OhC/g6COPGnpsKInbh6E0H2fziWK0kFas7aMi7idC9vpj63KeQcl4MA980
-60RKJKK9pckmcKTqKcpVaeQJFJFeztuXRJMg1jNd2RIFXLgbtp80Mw0vCsXF
-y8D4QhqYftEKJt0x7rX7D0hIeJLlPQ70ZOwnkGH/WbASCPGms+C94QGFFZ1Y
-JYzmtFya0nGWD3fXHKA/XtseKzo4lLvMhwD9Su9Ixq/c5fl5kSRq1IrAyXJQ
-HuyDlBSTfUtAdd2ESc4mHRrMfPGl4gOZSDng9Q5iV+n+xEz28nDZlLIdqmiM
-Ih1lb7KDraWrZwY0X+kFy+fQCHq9i6ufI+WOeXF7p5zVeDvrdkXY6HGJmEnS
-xX/lKsgMlN+Cf1uRKM6uKQ4IuTvsfLirMdjVuPtS9IuyYyVopsuVLm5Np+m7
-PZfSfzOqLupuvVw39ZwzFVeSNjwNXyuCub6pidY2POT8cM2FM6tcwCjoGWOT
-Fg5sj8qrDeWPJVgTogKmgtnArp/1u3o954dmyBPArmLZxqBv7sRzWencHaJy
-pfLIluLRLtb57cFRv7D/TKTzMNYQ2U2lGQaWiL1RpgFs8gDrGSlOciITdKqZ
-A2TtL0XztDdJDRMkGuIY37xrndSiwf6B0OJlHtyCiawRt6gM1gaaKJft6+1S
-5q96YikfQIZH+0AbkBb5tHLZbrmJXUt0xEx5kjXjkLnieFgeaHLeWiPkEqBU
-q+qrerGEt78VSDjexQSWaQWA6+1KTMpUZho+qQhuiPJQX0oRKt/PFBCUBFWr
-TuX6myLhzHnrx3G71695Fah32Iwwwa9A4AR/E6pBurEm+XW67cmlJFtFWuxC
-Wu+pgCI1kuO/6KQEhr4q9ReIjDR11zVX8UPzsaQaNhTBpux7BtKvLuPWa9aT
-ZTzKmq2DEiYbMJ9gsnXmCkItAtlkZdaGzg/NCFFi6VPvfaU78lY8b6gIfeM7
-MyiCcNFuc8dn94VH2isvkTHf/zxcPuNnnD9zRm1yPVX/79oZVbaQjKJ1psZQ
-YIlHQdKtrhbz+bI5b99z1eKT3vVSSPuor9i2lS+S1hpHNdDXIbJrvmUQ5GTT
-4ej90bE4I0KRKyhMIn0mGmh053KxbDg7KZXbyLkLznfTy5W2BrIX2Thl0oj5
-ur0G8BAbuuFiu2bV0rDAe2htOiM4rGyiI8jIIEB1D3LIQvrFnFtonyYuZc/K
-doabiiAmcZ9HJYGGtAYc4LyhR9EXKlcXAENTWF1iDh6Wpt2RjXJwVeaLZCao
-NyYGFlziY+t
```

Comment:
The rest of these are insufficiently clear-cut for me to clarify without hearing from @ekr